### ISSUE TITLE
Feature Request: Possible to save log in wigle compatible csv format?

### ISSUE THREAD

**Issue by lozaning** [kkonradpl/mtscan#9]
Would it be possible to add the functionality to export or save my wifi scan data and gps info into a log file thats in the wigle compatible csv format? https://api.wigle.net/csvFormat.html

**Comment by kkonradpl** [comment]
I've been discussing this on Wigle IRC lately. Altitude and accuracy are mandatory values which are used for data verification (discarding invalid entries). This is currently missing in MTscan, and because of that, there is no straightforward way to export any existing MTscan logs.

I'll implement these additional values and the export function (hopefully) soon.

**Comment by lozaning** [comment]
I tried piecing together my own pr to implement this, but got stuck on getting the actual kind of encryption a given network has (I only got a P or no P depending on if there was any encryption). Where you able to figure out how to query for that info somehow? 
It looked like the second generation of the microtik wireless management software supports returning this, but the AP I have dont run that since they're pretty old.

**Comment by kkonradpl** [comment]
I don't think that it is possible to obtain this information from MikroTik scanner. However it should be possible to implement it in the sniffer mode (parsing beacons and probe responses).

During last weekend I found some time to implement support for altitude & accuracy values and test it: cda0814ceab113bfbffa3692534e525a2c76af64
(tested with gpsd only, untested on Windows with sensor API, but altitude and accuracy support was already implemented)

I also created an CSV exporter: 4eac459ef4ac7f08dde1cd5d545bfa5f6396e69f
But this should be considered as experimental and untested feature. Use the export button from toolbar and select WiGLE (*.csv) instead of HTML. Currently there is no support for AuthMode at all.
